api_docs/unmerged.d/ZF-737924.md

```diff
@@ -0,0 +1 @@
+* [`POST /api/create-channel`](/api/create-channel): Added a new endpoint for creating a new channel.
```

Comment:
The first part of this is wrong. This is a mistake that could easily be avoided by taking the time to look at `api_docs/changelog.md` and seeing how other entries are written. The `POST ...` part always has the URL for the endpoint, not some other random string.